src/modules/commander/HealthAndArmingChecks/checks/offboardCheck.cpp

```diff
@@ -47,9 +47,10 @@ void OffboardChecks::checkAndReport(const Context &context, Report &reporter)
 	offboard_control_mode_s offboard_control_mode;
 
 	if (_offboard_control_mode_sub.copy(&offboard_control_mode)) {
-		bool offboard_available = offboard_control_mode.position || offboard_control_mode.velocity
-					  || offboard_control_mode.acceleration || offboard_control_mode.attitude || offboard_control_mode.body_rate
-					  || offboard_control_mode.actuator;
+		bool data_is_recent = hrt_absolute_time() < offboard_control_mode.timestamp + 100_ms;
```

Comment:
Shouldnt the timeout come from `COM_OF_LOSS_T`?

Reply:
there's a hysteresis below for that

Reply:
Updated